src/manifest.rs

```diff
@@ -72,6 +72,8 @@ impl CargoManifest {
         let filename = self.package.name.replace("-", "_");
         let wasm_file = format!("{}_bg.wasm", filename);
         let js_file = format!("{}.js", filename);
```

Comment:
i think we can remove this now!

Reply:
Just wanna double-check this one. It looks like removing this would break an existing test (`it_creates_a_package_json_default_path()`, line 63 in the file). Do we not want this `js_file` to become the `"main"` attribute in `package.json`?